packages/scoutgame-ui/src/components/common/Card/BuilderCard/BuilderCardNftDisplay.tsx

```diff
@@ -44,13 +48,12 @@ export function BuilderCardNftDisplay({
 }) {
   const width = nftDisplaySize[size].width;
   const height = nftDisplaySize[size].height;
-
+  const { openModal } = useDeveloperInfoModal();
   return (
     <Box overflow='hidden' width={width} height={height} sx={{ backgroundColor: 'black.dark', borderRadius: '4px' }}>
       <CardActionArea
         disabled={disableProfileUrl}
-        LinkComponent={Link}
-        href={`/u/${path}`}
+        onClick={() => openModal(path)}
```

Comment:
I think it would be nice if we kept the href attributes so you could still right-click and open in a new tab

Reply:
good idea. updated